deploy/helm/kubernetes/templates/etcd-statefulset.yaml

```diff
@@ -37,6 +36,10 @@ spec:
       labels:
         app: {{ $fullName }}-etcd
     spec:
+      {{- if .Values.etcd.image.PullSecrets }}
+      imagePullSecrets:
+        {{- toYaml .Values.etcd.image.PullSecrets | nindent 10 }}
+      {{- end }}
```

Comment:
Just minor, this syntax might be a bit better:

```yaml
      {{- with .Values.etcd.image.PullSecrets }}
      imagePullSecrets:
        {{- toYaml . | nindent 10 }}
      {{- end }}
```

Reply:
see f805faa10c2fb7bd794c1aea7025c6e32bf06d40